core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -480,7 +480,8 @@ class KafkaServer(
         endpoint
     )
 
-    val jmxPort = System.getProperty("com.sun.management.jmxremote.port", "-1").toInt
+    val jmxPort = System.getProperty("com.sun.management.jmxremote.port", "
+    ").toInt
```

Comment:
This should be typo. It caused **compile error**!

Reply:
Yeah, this was an unintentional change in the last commit. I'll fix it along with the test issues.